renovate.json

```diff
@@ -12,7 +12,7 @@
       "automerge": true,
       "rangeStrategy": "pin",
       "schedule": [
-        "before 3am on Monday"
+        "before 3am on Tuesday"
```

Comment:
Why Tuesday?

Reply:
Lorenzo requested it because some are off on Mondays :D https://github.com/smg-automotive/renovate-config/pull/2#discussion_r949230739